main.go

```diff
@@ -111,6 +124,11 @@ func getCSS(rw http.ResponseWriter, req *http.Request, args map[string]string) {
 	}
 }
 
+func init() {
+	flag.Var(&attrs, "a", "period separated list of attrs to be assigned to nodes (one per node, in order) e.g. -a=ssd.ssd.ssd.hdd")
+	flag.Var(&localities, "l", "period separated list of localities to be assigned to nodes (one per node, in order) e.g. -l=l1=a,l2=a.l1=a,l2=b.l1=b,l1=a")
```

Comment:
Yeah, I couldn't use semicolons, we already use colons and commas.

It's already setup to be specified multiple times. 

I just wanted it to be super quick to define in the command line, but I've made the change as you suggested and internally it now uses a map instead of a slice.